src/System.CommandLine.Suggest/dotnet-suggest-shim.bash

```diff
@@ -24,6 +24,11 @@ _dotnet_bash_complete()
     fi
 }
 
-complete -F _dotnet_bash_complete `dotnet-suggest list`
+_dotnet_bash_register_complete()
+{
+    local IFS=$'\n'
+    complete -F _dotnet_bash_complete `dotnet-suggest list`
+}
+_dotnet_bash_register_complete
 export DOTNET_SUGGEST_SCRIPT_VERSION="1.0.0"
```

Comment:
Please update the script version to `1.0.1`.

Reply:
Done